service/storage_service.cc

```diff
@@ -1056,6 +1049,49 @@ class topology_coordinator {
         co_return std::pair{gen_uuid, std::move(gen_mutations)};
     }
 
+    // Broadcasts all mutations returned from `prepare_new_cdc_generation_data` except the last one.
```

Comment:
The commit message and this comment need to be updated

Reply:
Fixed.